web/load.js

```diff
@@ -111,7 +111,7 @@
 
     extjsVersion = '6.2.0';
     fontAwesomeVersion = '4.7.0';
-    olVersion = '3.19.1';
+    olVersion = '3.20.0';
```

Comment:
Updated.
I've seen it. So desirable feature and example for API. 
Need to share `DeviceImajes.js` library somehow between "simple" and full